examples/knowledge_generation_using_nemotron/utils.py

```
@@ -0,0 +1,117 @@
+    generate_knowledge_qa_dataset
+)
+from datasets import concatenate_datasets
+from sdg_hub.prompts import PromptRegistry
+from sdg_hub.blocks import BlockRegistry, Block
+from datasets import Dataset
+import re
+from typing import List
+
+def _conv_pretrain(rec, tokenizer):
+    if tokenizer is not None:
+        rec['unmask'] = True
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)